cmd/context/update-kubeconfig.go

```diff
@@ -47,39 +62,28 @@ func UpdateKubeconfigCMD() *cobra.Command {
 			ctx := context.Background()
 
 			// Run context command to get the Cfg into Okteto Context
-			if err := NewContextCommand().Run(ctx, &ContextOptions{}); err != nil {
+			if err := NewContextCommand(withKubeTokenController(kc.kubetokenController)).Run(ctx, &ContextOptions{}); err != nil {
```

Comment:
a bit confusing. Could be sustitute by?:
```go
ctxCommand := NewContextCommand()
ctxCommand.kubetokenController = kc.kubetokenController
```
or 
```go
ctxCommand := NewContextCommand(kc.kubetokenController)
```
what are the benefits of using your proposal?

Reply:
if it is an improvement, should we add a check for nil values?